custom/enikshay/integrations/nikshay/repeaters.py

```diff
@@ -26,10 +30,10 @@ def allowed_to_forward(self, episode_case):
         # When case property episode.episode_pending_registration transitions from 'yes' to 'no',
         # and (episode.nikshay_registered != 'true'  or episode.nikshay_id != '')
         episode_case_properties = episode_case.dynamic_case_properties()
-        (not episode_case_properties['nikshay_registered'] and
-         not episode_case_properties['nikshay_id'] and
-         episode_pending_registration_changed(episode_case)
-         )
+        return (not episode_case_properties.get('nikshay_registered', False) and
```

Comment:
i think this should be `episode_case_properties.get('nikshay_registered', False) != 'true'`, since that property is stored as a string.